bors.toml

```diff
@@ -1,5 +1,8 @@
 status = [
-    "wasmerio.wasmer"
+    "wasmerio.wasmer",
```

Comment:
I think we don't need this status check. This was the status name of tests in Azure